include/wlr/types/wlr_xdg_foreign_v2.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * This an unstable interface of wlroots. No guarantees are made regarding the
+ * future consistency of this API.
+ */
+#ifndef WLR_USE_UNSTABLE
+#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
+#endif
+
+#ifndef WLR_TYPES_WLR_XDG_FOREIGN_V2_H
+#define WLR_TYPES_WLR_XDG_FOREIGN_V2_H
+
+#include <wayland-server-core.h>
+#include <wlr/types/wlr_xdg_foreign_registry.h>
+
+struct wlr_xdg_foreign_v2 {
+	struct {
+		struct wl_global *global;
+		struct wl_list resources;
+		struct wl_list clients;
```

Comment:
Do we really need per-client lists? Can't we just have directly a list of `wlr_xdg_exported_v2`/`wlr_xdg_imported_v2`?

Reply:
yeah the list is actually of wlr_xdg_exported_v2 not of clients, I think `objects` would be a better name, or what do you think?